src/main/webapp/snippets/workflow.js

```diff
@@ -62,35 +62,24 @@ snippet for++\n\
 	for (int ${1:i} = 0; $1 < ${2:Things}.length; $1++) {\n\
 		${3:$2[$1]}$0\n\
 	}\n\
-# stage\n\
-snippet stage\n\
-	stage '${1}'\n\
-# node\n\
-snippet node\n\
-	node {\n\
-	    ${1}\n\
-	}\n\
-# node-l ('label') \n\
-snippet node-l\n\
-	node ('${1}') {\n\
-	    \n\
-	}\n\
 ";
 
 // TODO: Would be nice to get this from the backend via an AJAX call.
 // With each step contributing 1 or more snippets of code
 var workflowSnippets =
 "# stage\n\
 snippet stage\n\
-	stage '${1}'\n\
+	stage('${1}')\n\
+	    ${2}\n\
+	}\n\
 # node\n\
 snippet node\n\
 	node {\n\
 	    ${1}\n\
 	}\n\
 # node-l ('label') \n\
 snippet node-l\n\
-	node ('${1}') {\n\
+	node('${1}') {\n\
```

Comment:
Bringing snippet style in line with what `Snippetizer` and most docs would suggest.